src/routes/payments.py

```
@@ -96,10 +95,49 @@
     request: Request,
     settings: BaseAppSettings = Depends(get_settings),
     db: AsyncSession = Depends(get_db),
+    email_sender: EmailSenderInterface = Depends(
+        get_accounts_email_notificator
+    ),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)